EventFilter/L1TRawToDigi/plugins/implementations_stage2/GMTCollections.cc

```diff
@@ -19,6 +19,12 @@ namespace l1t {
       event_.put(std::move(imdMuonsEMTFPos_), "imdMuonsEMTFPos");
       event_.put(std::move(imdMuonsOMTFNeg_), "imdMuonsOMTFNeg");
       event_.put(std::move(imdMuonsOMTFPos_), "imdMuonsOMTFPos");
+
+      event_.put(std::move(regionalMuonShowersEMTF_), "EMTF");
+      event_.put(std::move(muonShowers_[0]), "MuonShower");
+      for (int i = 1; i < 6; ++i) {
```

Comment:
Could this "6" be made a named constant (if it cannot be taken from somewhere else) to be used also in https://github.com/cms-sw/cmssw/pull/35627/files#diff-b58cb5d2c0f29e48000d3f6f50516977599983b3142acf4599330e59d0268067R74 ?

Reply:
Hi Andrea,

True, fair point. I've replaced it by a constant. It's the number of uGT boards we have installed so it's unlikely to be changed however it's now a bit more explicit.

Cheers,
Dinyar